fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/LogicalMetricColumn.java

```diff
@@ -23,27 +21,22 @@ public LogicalMetricColumn(String name, LogicalMetric metric) {
     }
 
     /**
-     * Getter for a logical metric.
+     * Constructor.
      *
-     * @return logical metric
+     * @param metric  The logical metric
      */
-    public LogicalMetric getLogicalMetric() {
-        return this.metric;
+    public LogicalMetricColumn(LogicalMetric metric) {
+        super(metric.getName());
+        this.metric = metric;
```

Comment:
I know there was a reason this was like this, I was in the middle of a refactor and got distracted, I think.